spec/services/doc_auth/lexis_nexis/requests/true_id_request_spec.rb

```diff
@@ -100,127 +100,44 @@
         end
       end
     end
-
-    def include_liveness_expected
-      FeatureManagement.idv_allow_selfie_check? &&
-        liveness_checking_required
-    end
   end
 
-  context 'with liveness_checking_enabled as false' do
```

Comment:
This diff becomes a little confusing, but tl;dr I removed this context. I kept the _content_ of `context 'with liveness_checking_enabled as true'` but removed the context block itself because that's now always the case.